src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -3,6 +3,30 @@
 <html>
 	<head>
 		<script>
+
+			window.gitpodIDEErrors = []
+
+			const onVsCodeWorkbenchError = (event) => {
+				const flags = new Map();
+
+				// We take a look at what is the resource that was attempted to load;
+				const resourceSource = event?.target?.getAttribute('src') || event?.target?.getAttribute('href');
+
+				// If the event has a `target`, it means that it wasn't a script error
+				if (resourceSource) {
+					flags.set('resource', 'vscode-web-workbench');
+					flags.set('error', 'LoadError');
+				} else {
+					flags.set('error', 'Unknown');
+				}
+				window.gitpodIDEErrors.push(flags)
+			}
+
+			window.onerror = onVsCodeWorkbenchError;
+
+			// Upon load, supervisor frontend can take over
+			winndow.onload = () => window.removeEventListener('error', onVsCodeWorkbenchError);
```

Comment:
We should always use `addEventListener` not assignment. Assignment can be overridden by someone else.

Reply:
Removed